scripts/generate_docs.sh

```diff
@@ -0,0 +1,56 @@
+ #!/usr/bin/env bash
+
+set -eo pipefail
+
+mkdir -p ./tmp-swagger-gen
+
+# move the vendor folder to a temp dir so that go list works properly
+temp_dir="f29ea6aa861dc4b083e8e48f67cce"
```

Comment:
What about using `mktemp -d` - this creates a unique temporary directory and ensures there are no collisions. This also makes the script re-runnable by doing away with having to check if the temp directory already exists on subsequent runs.

I found this stackoverflow post about it: https://stackoverflow.com/a/4632032

Reply:
The `tmp-swagger-gen` needs to be hardcoded for `./docs/config.json` to work properly.
I do like that mktemp approach for the vendor hack though, thank you!

Fixed in https://github.com/noria-net/noria/pull/11/commits/98aca435addf76ada0ac6f3e361c028907d734f1